pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/visitor/JavaCodeWriter.java

```diff
@@ -2578,19 +2583,18 @@ private static class CodeTemplate implements CodeToResolve {
             assert template != null;
 
             this.astNode = astNode;
+            this.plcsqlPos = plcsqlPos;
 
             int plcsqlLine = plcsqlPos[0];
             int plcsqlColumn = plcsqlPos[1];
 
-            // used: plcsqlLineColumn, template, substitutions
             if (plcsqlLine < 0 && plcsqlColumn < 0) {
-                this.plcsqlLineColumn = null; // do not mark code range in this case
+                // OK
             } else {
                 assert plcsqlLine > 0 && plcsqlColumn > 0
                         : String.format(
                                 "%s - line and column numbers of code templates must be positive integers: (%d, %d)",
                                 astNode, plcsqlLine, plcsqlColumn);
-                this.plcsqlLineColumn = String.format("%d,%d", plcsqlLine, plcsqlColumn);
             }
```

Comment:
If this block is only for assert, the if-statement seems redundant.

Reply:
It needs to be simplified. Thank you.